service/frontend/templates/accesscontrolled.tmpl

```diff
@@ -81,6 +81,11 @@ func New{{$Decorator}}(handler {{$.Interface.Type}}, resource resource.Resource,
 
 {{range $method := .Interface.Methods}}
 func (a *{{$decorator}}) {{$method.Declaration}} {
+	{{- if eq $method.Name "DescribeCluster" }}
+	defer func() {
+		log.CapturePanic(recover(), a.GetLogger(), &err)
+	}()
+	{{- end }}
```

Comment:
nit: maybe add a panic wrapper instead of merging two in the same place. The issue was handler recover logic is not the most outside wrapper and thus cannot capture panic in the wrappers. 
The ultimate solution is to put a panic wrapper in the most outside